+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Tue Dec  6 21:59:58 2022 - INFO  - Start environmental preparation.
+ DNF_INSTALL avahi
+ pkgs=avahi
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs avahi --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpw5ad7j6e
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpw5ad7j6e
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Tue Dec  6 22:01:34 2022 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Tue Dec  6 22:01:34 2022 - INFO  - Start testing...
+ test_execution avahi-daemon.service
+ service=avahi-daemon.service
++ date '+%Y-%m-%d %T'
+ log_time='2022-12-06 22:01:34'
+ test_restart avahi-daemon.service
+ service=avahi-daemon.service
+ systemctl restart avahi-daemon.service
A dependency job for avahi-daemon.service failed. See 'journalctl -xe' for details.
+ CHECK_RESULT 1 0 0 'avahi-daemon.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='avahi-daemon.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'avahi-daemon.service restart failed'
+ LOG_ERROR 'avahi-daemon.service restart failed'
+ message='avahi-daemon.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'avahi-daemon.service restart failed'
Tue Dec  6 22:01:35 2022 - ERROR - avahi-daemon.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Tue Dec  6 22:01:35 2022 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status avahi-daemon.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'avahi-daemon.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='avahi-daemon.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'avahi-daemon.service restart failed'
+ LOG_ERROR 'avahi-daemon.service restart failed'
+ message='avahi-daemon.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'avahi-daemon.service restart failed'
Tue Dec  6 22:01:42 2022 - ERROR - avahi-daemon.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Tue Dec  6 22:01:42 2022 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop avahi-daemon.service
Warning: Stopping avahi-daemon.service, but it can still be activated by:
  avahi-daemon.socket
+ CHECK_RESULT 0 0 0 'avahi-daemon.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='avahi-daemon.service stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status avahi-daemon.service
+ grep 'Active: inactive'
     Active: inactive (dead)
+ CHECK_RESULT 0 0 0 'avahi-daemon.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='avahi-daemon.service stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl start avahi-daemon.service
A dependency job for avahi-daemon.service failed. See 'journalctl -xe' for details.
+ CHECK_RESULT 1 0 0 'avahi-daemon.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='avahi-daemon.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'avahi-daemon.service start failed'
+ LOG_ERROR 'avahi-daemon.service start failed'
+ message='avahi-daemon.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'avahi-daemon.service start failed'
Tue Dec  6 22:01:49 2022 - ERROR - avahi-daemon.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Tue Dec  6 22:01:50 2022 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status avahi-daemon.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'avahi-daemon.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='avahi-daemon.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'avahi-daemon.service start failed'
+ LOG_ERROR 'avahi-daemon.service start failed'
+ message='avahi-daemon.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'avahi-daemon.service start failed'
Tue Dec  6 22:01:56 2022 - ERROR - avahi-daemon.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Tue Dec  6 22:01:57 2022 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled avahi-daemon.service
+ service=avahi-daemon.service
++ systemctl is-enabled avahi-daemon.service
+ state=enabled
+ '[' enabled == enabled ']'
++ systemctl disable avahi-daemon.service
++ grep Removed
++ awk '{print $2}'
++ awk '{print substr($0,1,length($0)-1)}'
+ symlink_file='/etc/systemd/system/multi-user.target.wants/avahi-daemon.service
/etc/systemd/system/dbus-org.freedesktop.Avahi.service
/etc/systemd/system/sockets.target.wants/avahi-daemon.socket'
+ find /etc/systemd/system/multi-user.target.wants/avahi-daemon.service /etc/systemd/system/dbus-org.freedesktop.Avahi.service /etc/systemd/system/sockets.target.wants/avahi-daemon.socket
find: ‘/etc/systemd/system/multi-user.target.wants/avahi-daemon.service’: No such file or directory
find: ‘/etc/systemd/system/dbus-org.freedesktop.Avahi.service’: No such file or directory
find: ‘/etc/systemd/system/sockets.target.wants/avahi-daemon.socket’: No such file or directory
+ CHECK_RESULT 1 0 1 'avahi-daemon.service disable failed'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='avahi-daemon.service disable failed'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ systemctl enable avahi-daemon.service
Created symlink /etc/systemd/system/dbus-org.freedesktop.Avahi.service → /usr/lib/systemd/system/avahi-daemon.service.
Created symlink /etc/systemd/system/multi-user.target.wants/avahi-daemon.service → /usr/lib/systemd/system/avahi-daemon.service.
Created symlink /etc/systemd/system/sockets.target.wants/avahi-daemon.socket → /usr/lib/systemd/system/avahi-daemon.socket.
+ find /etc/systemd/system/multi-user.target.wants/avahi-daemon.service /etc/systemd/system/dbus-org.freedesktop.Avahi.service /etc/systemd/system/sockets.target.wants/avahi-daemon.socket
/etc/systemd/system/multi-user.target.wants/avahi-daemon.service
/etc/systemd/system/dbus-org.freedesktop.Avahi.service
/etc/systemd/system/sockets.target.wants/avahi-daemon.socket
+ CHECK_RESULT 0 0 0 'avahi-daemon.service enable failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='avahi-daemon.service enable failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ journalctl --since '2022-12-06 22:01:34' -u avahi-daemon.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
Dec 06 22:01:34 anolis-riscv systemd[1]: Dependency failed for avahi-daemon.service - Avahi mDNS/DNS-SD Stack.
Dec 06 22:01:34 anolis-riscv systemd[1]: avahi-daemon.service: Job avahi-daemon.service/start failed with result 'dependency'.
Dec 06 22:01:49 anolis-riscv systemd[1]: Dependency failed for avahi-daemon.service - Avahi mDNS/DNS-SD Stack.
Dec 06 22:01:49 anolis-riscv systemd[1]: avahi-daemon.service: Job avahi-daemon.service/start failed with result 'dependency'.
+ CHECK_RESULT 0 0 1 'There is an error message for the log of avahi-daemon.service'
+ actual_result=0
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of avahi-daemon.service'
+ '[' -z 0 ']'
+ '[' 1 -eq 0 ']'
+ test 0x == 0x
+ test -n 'There is an error message for the log of avahi-daemon.service'
+ LOG_ERROR 'There is an error message for the log of avahi-daemon.service'
+ message='There is an error message for the log of avahi-daemon.service'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'There is an error message for the log of avahi-daemon.service'
Tue Dec  6 22:02:03 2022 - ERROR - There is an error message for the log of avahi-daemon.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 28'
+ message='../common/common_lib.sh line 28'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 28'
Tue Dec  6 22:02:03 2022 - ERROR - ../common/common_lib.sh line 28
+ return 0
+ systemctl start avahi-daemon.service
A dependency job for avahi-daemon.service failed. See 'journalctl -xe' for details.
+ sed -i 's\ExecStart=/usr/sbin/avahi-daemon\ExecStart=/usr/sbin/avahi-daemon --debug\g' /usr/lib/systemd/system/avahi-daemon.service
+ systemctl daemon-reload
+ systemctl reload avahi-daemon.service
avahi-daemon.service is not active, cannot reload.
+ CHECK_RESULT 1 0 0 'avahi-daemon.service reload failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='avahi-daemon.service reload failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'avahi-daemon.service reload failed'
+ LOG_ERROR 'avahi-daemon.service reload failed'
+ message='avahi-daemon.service reload failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'avahi-daemon.service reload failed'
Tue Dec  6 22:02:06 2022 - ERROR - avahi-daemon.service reload failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_avahi-daemon.sh line 35'
+ message='oe_test_service_avahi-daemon.sh line 35'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_avahi-daemon.sh line 35'
Tue Dec  6 22:02:07 2022 - ERROR - oe_test_service_avahi-daemon.sh line 35
+ return 0
+ systemctl status avahi-daemon.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'avahi-daemon.service reload causes the service status to change'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='avahi-daemon.service reload causes the service status to change'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'avahi-daemon.service reload causes the service status to change'
+ LOG_ERROR 'avahi-daemon.service reload causes the service status to change'
+ message='avahi-daemon.service reload causes the service status to change'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'avahi-daemon.service reload causes the service status to change'
Tue Dec  6 22:02:08 2022 - ERROR - avahi-daemon.service reload causes the service status to change
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_avahi-daemon.sh line 37'
+ message='oe_test_service_avahi-daemon.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_avahi-daemon.sh line 37'
Tue Dec  6 22:02:08 2022 - ERROR - oe_test_service_avahi-daemon.sh line 37
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Tue Dec  6 22:02:09 2022 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Dec  6 22:02:09 2022 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Tue Dec  6 22:02:10 2022 - INFO  - start environment cleanup.
+ sed -i 's\ExecStart=/usr/sbin/avahi-daemon --debug\ExecStart=/usr/sbin/avahi-daemon\g' /usr/lib/systemd/system/avahi-daemon.service
+ systemctl daemon-reload
+ systemctl reload avahi-daemon.service
avahi-daemon.service is not active, cannot reload.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpw5ad7j6e ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpw5ad7j6e
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Tue Dec  6 22:02:24 2022 - INFO  - Finish environment cleanup!
